src/org/ods/orchestration/usecase/DocGenUseCase.groovy

```diff
@@ -146,9 +147,77 @@ abstract class DocGenUseCase {
         return "${documentType}-${result}-${version}-${build}".toString()
     }
 
+    @SuppressWarnings(['AbcMetric'])
+    Map resurrectAndStashDocument(String documentType, Map repo, boolean stash = true) {
+        if (!repo.data?.odsBuildArtifacts || !repo.data.odsBuildArtifacts?.deployments) {
+            return [found: false]
+        }
+        String resurrectedBuild
+        if (!!repo.data.odsBuildArtifacts.resurrected) {
+            resurrectedBuild = repo.data.odsBuildArtifacts.resurrected
```

Comment:
I'm not sure we can resurrect and reuse old documents/builds for official releases into dev, qa and prod. Since the document metadata content matters, it has to be the correct one. And if it was created in WIP/developer preview, this is an issue.

Reply:
see above